src/rgw/rgw_lc.cc

```diff
@@ -1422,6 +1422,16 @@ std::string rgwlc_s3_expiration_header(
       return hdr;
   } /* catch */
 
+  /* dump tags at debug level 16 */
+  RGWObjTags::tag_map_t obj_tag_map = obj_tagset.get_tags();
```

Comment:
@mattbenjamin move this into internal scope?

Reply:
probably, will look again

Reply:
I don't think there is anywhere obvious to move obj_tag_map decl--this is just a lexical reference that was previously used only in expressions prior to the change--I did that so that my debug prints could use it, but it doesn't make any difference in efficiency to move it?